USemLog.uplugin

```diff
@@ -52,6 +52,10 @@
 		{
 			"Name": "libmongo",
 			"Enabled": true
+		},
```

Comment:
here as well, by default the plugin should not depend on this

Reply:
I'll be commenting this out and the other one.

Reply:
Should I disable UUtils and UMC also? @andreihaidu

Reply:
@joserochh no, those can stay since they are in the "robcog ecosystem" once we move the slicing one as well from the student project we can also leave that by default.